multi-cloud/README.md

```diff
@@ -415,15 +447,27 @@ kubectl get pods -n kubeflow -w | grep sk-train
 
 These will hopefully show a successfully running set of pods / job.
 
+You should follow along as the Pod goes from `Pending` --> `ContainerCreating` --> `Running` --> `Completed` using the get kubectl pods command.
+
+In addition, you can use the `~/argo get [WORKFLOW_NAME] -n kubeflow`, `~/argo list [WORKFLOW_NAME] -n kubeflow`, `~/argo describe [WORKFLOW_NAME] -n kubeflow`, and `~/argo delete [WORKFLOW_NAME] -n kubeflow` to interact with workflows that you create.
+
 ### Serve the Model
 
 Once training has finished, we can serve it with:
 
 ```
 cd $EXAMPLE_SELDON/workflows
-~/argo submit serving-sk-mnist-workflow.yaml -n kubeflow -p deploy-model=true
+~/argo submit serving-sk-mnist-workflow.yaml -n kubeflow
```

Comment:
also wrong.

Reply:
that's my bad I missread this last night